topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md

```diff
@@ -0,0 +1,159 @@
+
+### In this topic
+
+This topic contains the following sections:
+
+-   [Configure the `gridFeatureOptions` settings](#gridfeatureoptions_configure)
+-   [Configure exporting igHierarchicalGrid](#hierarchical_data)
+-   [Configure columns that will not be exported](#configure_columns)
+-   [Configure styling](#configure_styling)
+-   [Attach to callbacks (events)](#callbacks)
+-   [Display an overlay while exporting](#exporting_overlay)
+
+### Required background
+- [igGridExcelExporter Overview](iggridexcelexporter-overview.html "igGridExcelExporter Overview") - General information on the `igGridExcelExporter` control.
+
+
+### <a id="gridfeatureoptions_configure"></a>Configure the `gridFeatureOptions` settings
+
+Data and layout manipulations caused by the `Filtering`, `Hiding`, `Paging`, `Sorting`, `Summaries` and `Column Fixing` features of the `igGrid` can be reflected in the exported Excel Worksheet. The setting for each feature is passed within the [`gridFeatureOptions`](%%jQueryApiUrl%%/ig.gridexcelexporter#options:settings.gridFeatureOptions) object.
+All of these are enumerations and the `sorting`, `summaries` and `columnfixing` have only options - `"none"` or `"applied"`.  
+
+|Setting | Property | Description
+| -------------------| ------------------- |----------- 
+Configure Sorting | [sorting](%%jQueryApiUrl%%/ig.gridexcelexporter#options:settings.gridFeatureOptions.sorting) | Accepts values `"none"` or `"applied"`. When `"none"` Sorting is not applied in the Excel. When `"applied"` Sorting is applied in the Excel Worksheet.
+
+Code sample:
+
+```javascript
+$.ig.GridExcelExporter.exportGrid($(".selector"), {
+    gridFeatureOptions: {
+        // Fixed `igGrid` columns will be fixed in the excel worksheet
+        columnfixing: "applied",
+        // summaries will not be calculated in the worksheet
+        summaries: "none",
+        // Sorted `igGrid` columns will be sorted in the excel worksheet
+        sorting: "applied"
+    }
+});
+```
+
+Different options are available for configuring the `filtering`, `paging` and `hiding`. For example, we can opt to work with the visible `igGrid` columns only and do not export the hidden ones by setting `hiding: "visibleColumnsOnly"`. The other possible options for hiding are still `"none"` and `"applied"`. Setting `paging` and `filtering` is tricky as both features manipulate the data and different combinations will bring different results. For `paging` we can set either `"currentPage"` that will export only the current `igGrid` page, or `"allRows"` correspondingly. `filtering` can take values `"none"`, `"applied"`, or `"filteredRowsOnly"`. The difference between them - setting `"applied"` will make the `igGridExcelExporter` export all data and apply the same filter as applied in the `igGrid`, while setting `"filteredRowsOnly"` will export only the filtered records.
```

Comment:
You can put this information into the table above. Keep in mind that this topic is not to repeat the information from the API docs, but to add more value for the reader and actually describe actual end user scenarios. You can see how other "Configuration" topics are written. Most of them focus on user stories.